### PR TITLE
Base_oM: attribute descriptions added

### LINKED ISSUE #1528: Base_oM: add descriptions to the attributes
#### Description:
...as promised by myself under https://github.com/BHoM/BHoM/pull/1526

FYI @FraserGreenroyd

### PR DESCRIPTION
**Pull request by pawelbaran**
### Issues addressed by this PR

Closes #1528

### Test files

Descriptions only.

### CODE REVIEW

**Comment by pawelbaran** [comment]
@BHoMBot check compliance

File: BHoM/Attributes/FilePathAttribute.cs [hunk 1]
```diff
@@ -35,6 +35,7 @@ public class FilePathAttribute : ClassificationAttribute, IImmutable
         /**** Properties                                ****/
         /***************************************************/
 
+        [Description("Collection of file extensions allowed for a member this attribute corresponds to. Any extension is allowed if null.")]
```
Comment (FraserGreenroyd) [inline BHoM/Attributes/FilePathAttribute.cs:38]:
```suggestion
        [Description("Collection of file extensions allowed for a member this attribute corresponds to. Any extension is allowed if null or empty.")]
```

File: BHoM/Attributes/MultiOutputAttribute.cs [hunk 2]
```diff
@@ -23,23 +23,30 @@
...
     [AttributeUsage(AttributeTargets.Method, AllowMultiple = true)]
+    [Description("Attribute providing information about an output parameter of a multi-output method." +
+                 "\nTo be applied to all methods that return BH.oM.Base.Output types.")]
     public class MultiOutputAttribute : Attribute, IImmutable
     {
         /***************************************************/
         /**** Properties                                ****/
         /***************************************************/
 
+        [Description("Index of the output parameter this attribute corresponds to.")]
```
Comment (FraserGreenroyd) [inline BHoM/Attributes/MultiOutputAttribute.cs:40]:
```suggestion
        [Description("Zero-based index of the output parameter this attribute corresponds to.")]
```

**Review by FraserGreenroyd — COMMENTED** [review]
I like these, they all look good. I've made a couple of suggestions that might help future developers within the OS community, but I'm not terribly precious about them so if you disagree @pawelbaran just let me know and I'm happy to change this to approval 👍

**Review by FraserGreenroyd — APPROVED** [review]
All happy with descriptions, happy to merge

**Comment by FraserGreenroyd** [comment]
@BHoMBot check compliance
@BHoMBot check required

**Comment by FraserGreenroyd** [comment]
@BHoMBot this is a DevOps instruction. I am authorising dispensation to be granted on check ref. 14321807528

**Comment by FraserGreenroyd** [comment]
@BHoMBot check ready-to-merge

**Pull request merged**